hack/release.toml

```diff
@@ -75,7 +76,8 @@ Default `talosconfig` has `os:admin` role embedded in the certificate so that al
 Certificates with reduced set of roles can be created with `talosctl config new` command.
 
 When upgrading from Talos 0.10, RBAC is not enabled by default. Before enabling RBAC, generate `talosconfig` with
-`os:admin` role first to make sure that administrator still has access to the cluster when RBAC is enabled.
+`os:admin` role first to make sure that administrator still has access to the cluster when RBAC is enabled
+(usin `talosctl config new` command).
```

Comment:
```suggestion
(using `talosctl config new` command).
```

Reply:
how come I missed that, thanks, I'll repush that